### PR TITLE
Add cockpit icon.

### PR DESCRIPTION
**Pull request by martinpovolny**
Add cockpit icon. The same we have in operations UI.

### CODE REVIEW

File: client/app/states/services/details/details.html [hunk 1]
```diff
@@ -199,7 +199,7 @@ <h2 translate>Virtual Machines ({{ ::vm.vms.length }})</h2>
                   <i class="fa fa-html5 fa-lg"></i>
                 </button>
                 <button class="btn btn-default open-console-button" ng-if="item['supports_cockpit?'] && item.power_state == 'on'" tooltip="{{'Open Cockpit console for this VM'|translate}}" tooltip-placement="left">
-                  <i class="fa fa-html5 fa-lg"></i>
+                  <img src="/client/assets/images/cockpit.png"></img>
```
Comment (chriskacerguis) [inline client/app/states/services/details/details.html]: Were we using SVG for icons now?  @serenamarie125 thoughts?

Comment (martinpovolny) [inline client/app/states/services/details/details.html]: This is the same icon that we have in the operations UI and this icon is to be replaced by @serenamarie125 and her team once they have a font or SVG icon.

**Comment by chriskacerguis** [comment]
@martinpovolny Hey, moving forward can we please wait for any requested responses?

cc @chessbyte @dclarizio

**Comment by martinpovolny** [comment]
@chriskacerguis : Sure, that is the usual way how things are done. Except in this case the discussion you are started already happened between me, @dclarizio , @epwinchell and @serenamarie125 .

**Comment by chriskacerguis** [comment]
No worries, but I didn't know that and the PR was assigned to me :)

**Comment by chriskacerguis** [comment]
@martinpovolny sounds good, I was just more wanted to know from @serenamarie125 when that would be :)

**Comment by serenamarie125** [comment]
@martinpovolny can we add a screenshot ?  @chriskacerguis using font icons is the desired way to go, but when we have branded imagery they cannot be provided a monochrome font icons.

**Comment by chessbyte** [comment]
Euwe Backport details:
```sh
$ git log
commit f6a6d5cd05a9f048da196aad01de70577d4692b1
Author: Milan Zázrivec <mzazrivec@redhat.com>
Date:   Fri Sep 23 12:58:26 2016 +0200

    Merge pull request #177 from martinpovolny/cockpit_icon
    
    Add cockpit icon.
    (cherry picked from commit e73c22272d2a7eb33a4722d0737ebca605459edf)
```

**Pull request merged**